python/cudf/cudf_pandas_tests/third_party_integration_tests/dependencies.yaml

```diff
@@ -262,7 +262,7 @@ dependencies:
         packages:
           - pip
           - pip:
-              - ibis-framework[pandas]
+              - ibis-framework[pandas]<10.0.0
```

Comment:
This is fine for now, I'll remove it in #17945

Reply:
Yes, that was my intention behind this change.